Prioritize/README.md

```diff
@@ -70,7 +70,27 @@ on:
 
 ### Single Folder Log Publish
 If your pipeline publish does not allow for multi folder publishing like GitHub actions, then add the following script after your scan to copy all required folders to the whitesource folder. [AzureDevOps](../CI-CD#Azure-DevOps-Pipelines) is a good example where only single folder publishing is allowed.
+
+#### Linux based machines (Bash script)
 ```
 if [ -d "/tmp/whitesource*" ] ; then cp /tmp/whitesource* ./whitesource ; else echo "/tmp/whitesource* does not exist" ; fi
 if [ -d "/tmp/ws-ua*" ] ; then cp /tmp/whitesource* ./whitesource ; else echo "/tmp/ws-ua* does not exist" ; fi
 ```
+#### Windows based machines (Powershell script)
+```
+- task: PowerShell@2
+  inputs:
+    targetType: 'inline'
+    script: |
```

Comment:
remove lines 81-84 or mention that this script is for ADO and not just windows based machines

Reply:
added - thoughts?